demos/embeddings/watch-your-step-embeddings.ipynb

```diff
@@ -4,7 +4,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Node representation learning with Watch Your Step"
+    "# Node representation learning with Watch Your Step\n",
+    "\n",
+    "This notebook demonstrates how to use the StellarGraph implementation of [Watch Your Step](https://arxiv.org/abs/1710.09599)."
```

Comment:
This either needs to be a "summary", which is a block quote like

```suggestion
    "> This notebook demonstrates how to use the StellarGraph implementation of [Watch Your Step](https://arxiv.org/abs/1710.09599)."
```

Or it needs to be a separate cell, after the cloud runner (or just run `scripts/format_notebooks.py --default --overwrite demos/embeddings` after editing it).